packages/alfa-css/src/properties/font-weight.ts

```diff
@@ -0,0 +1,31 @@
+import { parse } from "@siteimprove/alfa-lang";
+import { FontWeightGrammar } from "../grammars/font-weight";
+import { Property } from "../types";
+
+export type FontWeight = Readonly<
+  // https://www.w3.org/TR/css-fonts/#font-weight-absolute-values
+  | {
+      type: "absolute";
+      value: 100 | 200 | 300 | 400 | 500 | 600 | 700 | 800 | 900;
+    }
+  | {
+      type: "relative";
+      value: "bolder" | "lighter";
+    }
+>;
```

Comment:
Can be simplified to:

```ts
export type FontWeight = number | "bolder" | "lighter";
```

Reply:
Not if there is a limited about of numbers to pick from.

Reply:
And do you want to discard the relativity attribute? I suppose we need it to infer whether or not to look at a parent element.

Reply:
See https://github.com/Siteimprove/alfa/pull/37#discussion_r210486158. We only need to know whether or not `font-weight` is numeric and then we look up the tree and apply the rules for `bolder` and `lighter` if it isn't.